deployment/network-operator/README.md

```diff
@@ -149,6 +149,40 @@ $ helm install -n network-operator --create-namespace --wait network-operator ./
 $ kubectl -n network-operator get pods
 ```
 
+#### Deploy Network Operator with Admission Controller
+
+The Admission Controller can be optionally included as part of the Network Operator installation process.  
+It has the capability to validate supported Custom Resource Definitions (CRDs), which currently include NicClusterPolicy and HostDeviceNetwork.  
+By default, the deployment of the admission controller is disabled. To enable it, you must set `admissionController.enabled` to `true`.  
```

Comment:
Please, add a note how to enable it for OCP

Reply:
OCP will be handled in a follow up PR